modules/ROOT/pages/_partials/task-flex-add-tls-context.adoc

```diff
@@ -82,3 +82,23 @@ To redeploy an API Instance:
 . Click *Runtime & Endpoint Configuration > Save & Apply*.
 
 //end::redeploy[]
+
+//tag::mTLS[]
+
+[[mTLS-context]]
+== mTLS Context
+Flex Gateway supports mutual authentication TLS (mTLS) for the inbound and outbound direction.
+
+In mTLS both the server and the client validates the others TLS certification. In normal TLS only the client validates the server's certificate. For Flex Gateway outbound traffic, the API upstream is the server and Flex Gateway is the client. For Flex Gateway inbound traffic, Flex Gateway is the server and the client is the client making requests to Flex Gateway. 
+
+Enable mTLS using the following UI:
```

Comment:
for outbound we need the keystore (to provide the certificate to use as a client)
for inbound we need the trustore (to validate the far side -client- certificate)